app/models/state_file_dependent.rb

```diff
@@ -78,17 +90,25 @@ def full_name
     parts.compact.join(' ')
   end
 
+  def months_in_home_for_pdf
+    months_in_home&.to_s || "<6"
+  end
+
+  def months_in_home_for_xml
+    months_in_home || 5
+  end
+
   def ask_senior_questions?
     return false if dob.nil?
-    senior? && months_in_home == 12 && ['PARENT', 'GRANDPARENT'].include?(relationship)
+    senior? && months_in_home == 12 && ['parent', 'grandParent', 'otherAncestorOfParent'].include?(relationship)
```

Comment:
do `stepParent`, `siblingOfParent`, `parentInLaw`, `fosterParent` not count for these scenarios?

Reply:
Dunno - forwarding to PM/program